cni/api/api.go

```diff
@@ -0,0 +1,15 @@
+package api
+
+import "net"
+
+type NetworkInterfaceInfo struct {
```

Comment:
This struct is very similar to IPConfigRequest, perhaps we could use the same, but `Request` suffix on that struct would be confusing in this usecase

Reply:
It is okay to use separate one, this contract is owned by CNI and later if we want to extend it then we dont need to touch other one